<!-- -->
<launch>

	 
	<node name="nuogeng_node" pkg="nuogeng" type ="nuogeng_node" output="screen" >
	
		<param name ="port_name"   value="/dev/ttyUSB0"/>
		<param name ="baudrate"    value="115200"/>
	</node>


</launch>












